python/jiminy_pywrap/include/jiminy/python/Jiminy.h

```diff
@@ -1164,20 +1284,28 @@ namespace python
                                    (bp::arg("self"), "model", "controller"))
                 .def("initialize", &PyEngineVisitor::initializeWithCallback,
                                    (bp::arg("self"), "model", "controller", "callback_handle"))
+
                 .def("reset", &Engine::reset,
-                              (bp::arg("self"), bp::arg("remove_forces")=false))
-                .def("set_state", &Engine::setState,
-                                  (bp::arg("self"), "x_init",
-                                   bp::arg("reset_random_generator")=false,
-                                   bp::arg("remove_forces")=false))
+                              (bp::arg("self"),
+                               bp::arg("reset_random_generator") = false,
```

Comment:
I would set this to `true` by default. 

Rational: calling `reset` then `simulate` twice should give the same result if the random seed in engine options is not changed.

Or this could be removed: such behavior (keeping the random seed) can be done using start/stop in a loop